### ISSUE TITLE
Footnote spacing without titlepage

### ISSUE THREAD

**Issue by folofjc** [dcpurton/biblatex-sbl#48]
In the file sbl-paper.tex, if the commane `\maketitle` is commented out, the footnote spacing is not correct. The attached file has this one line change (just change the extension from txt to tex). See the difference between footnote 1 and footnote 2. I also attached the pdf that I get.
[sbl-paper.txt](https://github.com/dcpurton/biblatex-sbl/files/950014/sbl-paper.txt)
[sbl-paper.pdf](https://github.com/dcpurton/biblatex-sbl/files/950015/sbl-paper.pdf)

For reference, I am using XeLaTeX and biber. I ran `xelatex`, followed by `biber`, followed by `xelatex` two more times to get the references right.

**Comment by dcpurton** [comment]
I'm pretty sure this is a duplicate of issue #49. The following code single spaces the footnote if `bidi` is not loaded:

```TeX
\documentclass{article}
\usepackage[style=verbose]{biblatex}
\usepackage{setspace}
\usepackage{bidi}
\doublespacing
\addbibresource{biblatex-examples.bib}
\begin{document}
Filler text \autocite{cotton}.
\end{document}
```

**Comment by folofjc** [comment]
Sorry, I spoke too soon. The fix in #49 does not fix this issue. Without `\maketitle` the footnotes are still double spaced.

**Comment by dcpurton** [comment]
This is not a duplicate. It's a bug in `sbl-paper.sty` caused by loading `setspace` before `bidi` is loaded.

It can be fixed by moving `RequirePackage{setspace}` to just before the `\doublespacing` command.

Getting packages loading in the write order with `bidi` is a pain :(

I'll push this change now.

**Comment by folofjc** [comment]
Yeah, I have had lots of pain with bidi over the years. Thanks for looking into it! I'll look for it in TeXLive.

**Comment by dcpurton** [comment]
These changes won't make it into TeXLive for a while. The `dev` branch is a bit unstable at the moment and there are some more changes I want to make.

Also, I have just discovered that moving `setspace` is no good! This causes other problems! Hyperlinks to footnotes break and apparently `footmisc` must be loaded after `setspace`. *sigh*

**Comment by dcpurton** [comment]
This is *sort of* fixed in `biblatex-sbl` 0.9. I no longer recommend using `polyglossia`. Use `babel` instead. The example `sbl-paper` document has been updated to do this. Since `bidi` is no longer loaded, you don't get any problems when not including a title page.

**Comment by dcpurton** [comment]
If anyone still needs to use `bidi` then I think this will help: https://tex.stackexchange.com/questions/479145/bidi-package-messing-up-line-spacing-in-xelatex/479197#479197

**Comment by folofjc** [comment]
Thanks, I'll look into it. I just also saw your comment from last year about `babel` vs `polyglossia`. I have not tried babel with Xelatex. I'll have to look into that as well.

**Comment by dcpurton** [comment]
@folofjc , I recommend `babel` with `lualatex` actually. The only exception is if any of your languages don't render well with `lualatex` and `xelatex` is required. Once `luatex` support for `harfbuzz` is finished this won't be a problem either.
